libcudacxx/include/cuda/std/__cmath/copysign.h

```diff
@@ -46,6 +47,11 @@ _CCCL_NODISCARD _LIBCUDACXX_HIDE_FROM_ABI float copysign(float __x, float __y) n
 #endif // !_CCCL_BUILTIN_COPYSIGN
 }
 
+_CCCL_NODISCARD _LIBCUDACXX_HIDE_FROM_ABI float copysign(float __x, float __y) noexcept
+{
+  return _CUDA_VSTD::copysignf(__x, __y);
+}
+
```

Comment:
In all other files we have 
```cpp
meow(float)
meowf(float)
```

I would love to have the same ordering everywhere